mockgen/parse.go

```diff
@@ -228,7 +253,7 @@ func (p *fileParser) parseFile(importPath string, file *ast.File) (*model.Packag
 
 	var is []*model.Interface
 	for ni := range iterInterfaces(file) {
-		if _, ok := p.excludeNamesSet[ni.name.String()]; ok {
+		if p.wantIface != nil && !p.wantIface(ni.name.String()) {
```

Comment:
should we error out here if `wantIface` was never found in the file? I don't think we're currently erroring out when we specify a nonexistent interface name.